pkg/openid4ci/openid4ci.go

```diff
@@ -97,14 +111,22 @@ func (i *Interaction) Authorize() (*AuthorizeResult, error) {
 // Relevant sections of the spec:
 // https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-7
 // https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html#section-8
-func (i *Interaction) RequestCredential(credentialRequestOpts *CredentialRequestOpts) ([]CredentialResponse, error) {
+func (i *Interaction) RequestCredential(credentialRequestOpts *CredentialRequestOpts) ([]CredentialResponse, error) { //nolint:funlen,lll
```

Comment:
Reason for the nolint?

Reply:
function becomes longer as we have multiline errors, and goes beyond line limits.  and I don't want to mess lint comment with the function comment so I added it to the end of the line so it required 'lll'. I don't have a good solution here.

Reply:
done

Reply:
I see. I think we might end up with many functions requiring `nolint:funlen` in that case. I don't really have an immediate solution either. If this becomes common, we might consider disabling funlen repo-wide @rolsonquadras